assembly/src/parsers/io_ops/local_ops.rs

```diff
@@ -24,12 +24,12 @@ pub fn parse_push_local(
 
 /// Pops the top element off the stack and saves it at the specified local procedure memory index as
 /// [top_element, 0, 0, 0].
-pub fn parse_pop_local(
+pub fn parse_store_local(
```

Comment:
We can actually do this operation much more efficiently now using `mstore` operation (instead of using `mstorew` + some stack manipulation).